addons/website/static/src/less/website.snippets.less

```diff
@@ -53,9 +53,32 @@ table.table_desc tr td {
 }
 
 // Other snippet styles
+.s_feature_grid p {
+    overflow: hidden;
```

Comment:
Mmh ok. I think this should work with `overflow: auto;` which is less "dangerous". Also, add a comment to explain the problem (or simply add "force the creation of a block formatting context").